src/parser.ts

```diff
@@ -133,6 +133,19 @@ const call = nodeMap(
       .skip(spaced(closeCmd('call'))))
 );
 
+const delCall = nodeMap(
+  S.DelCall,
+  P.string('{delcall')
+    .skip(P.whitespace)
+    .then(templateName),
+  optional(P.seq(P.whitespace, P.string('variant='))
```

Comment:
I think we can use the `attribute` parser here, and maybe combine it with `many` to handle the cases that @jbalsas mentioned.

Reply:
Thanks @mthadley. I'm getting acquainted with the parsimmon.